examples/inference-deployments/llama2/llama2_handler.py

```diff
@@ -0,0 +1,129 @@
+import copy
+import warnings
+from contextlib import nullcontext
+from enum import Enum
+from threading import Thread
+from typing import Any, Dict, List
+import os
+
+import deepspeed
+import torch
+from transformers import (AutoModelForCausalLM, AutoTokenizer,
+                          LlamaForCausalLM, LlamaConfig, LlamaTokenizer,
+                          TextIteratorStreamer, pipeline)
+
+class Llama2ModelHandler:
+
+    DEFAULT_GENERATE_KWARGS = {
+        'max_length': 256,
+        'use_cache': True,
+        'do_sample': True,
+        'top_p': 0.95,
+        'temperature': 0.8,
+    }
+
+    INPUT_KEY = 'input'
+    PARAMETERS_KEY = 'parameters'
+    MODEL_DTYPE = 'fp16'
+
+    def __init__(
+        self,
+        model_name_or_path: str,
+    ):
```

Comment:
Could we add a `max_output_tokens` parameter here, or in the go server? We'd then clip user `max_length` values in requests to this. 

This is a slow model and we'll probably need to limit it to globally to 128 tokens. Otherwise someone could send a super long request and tie up the entire server for >30 sec.